### DISCUSSION TITLE
OnlyM and Zoom Share Screen

### DISCUSSION THREAD

**Discussion by DKarpjr** [AntonyCorbett/OnlyM#443] [General]
Has anyone come up with a way to get Zoom share screen to work similar to how it functions with JWL...other than OBS or additional software?

Currently when sharing media from JWL we can use share screen in Zoom and click the JWL app and after clicking share, click play on the media and it works smoothly for both the displays and those on Zoom. 

But when we try to do the same steps with onlyM, it displays the media on the TV's fine but on Zoom it shows the onlyM app and no media.

So we've created a workaround by starting the media 1st and if a video pausing it.  Then use Zoom share screen and then select the secondary screen that the media is actually playing on. It works but the transitions aren't smooth and because of the time it takes to go between apps it can throw of the timing of images with the speakers prompts. So just wondering if I'm missing something in my setup that's preventing the same share screen functionality as with JWL.

**Comment by AntonyCorbett** [comment]
You must share the second screen; not the application.

**Reply by DKarpjr** [reply]
Ok. Thank you. That's what we've been doing just wasn't sure if there was a way to workaround that, that I wasn't aware of so figured I'd ask.  Thanks again.

**Reply by AntonyCorbett** [reply]
You can also share the OnlyM media window, but it’s probably easier to share the second screen

**Reply by mvpapen** [reply]
Hi there. We had very similar results. As soon your activate dual screen in Zoom , the share function is very unpredictable.
We solved it by using a capture card to capture hdmi out to 2 monitor. the capture card is connected via usb to the pc and shows as 2 camera in Zoom.
you can now toggle between the 2 cameras ( ALT+N) instead of sharing. Works great. We also use Focus mode to encourage the friends to stay in speaker view. After the meeting we turn off "Focus Mode"
